src/pages/en/core-concepts/astro-pages.md

```diff
@@ -5,17 +5,30 @@ description: An introduction to Astro pages
 i18nReady: true
 ---
 
-**Pages** are a special type of [Astro component](/en/core-concepts/astro-components/) that live in the `src/pages/` subdirectory. They are responsible for handling routing, data loading, and overall page layout for every HTML page in your website.
+**Pages** are special files that live in the `src/pages/` subdirectory. They are responsible for handling routing, data loading, and overall page layout for every HTML page in your website.
 
 ### File-based routing
 
-Astro leverages a routing strategy called **file-based routing.** Every `.astro` file in your `src/pages` directory becomes a page or an endpoint on your site based on its file path.
+Astro leverages a routing strategy called **file-based routing.**. Each page file in your `src/pages` directory becomes an endpoint on your site based on its file path.
 
 Write standard HTML [`<a>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) in your component template to link between pages.
 
-
 📚 Read more about [Routing in Astro](/en/core-concepts/routing/).
 
+### Supported page files 
+
+Astro supports the following file types in the `src/pages` directory:
+- [`.astro`](#astro-pages)
+- [`.md`](#markdown-pages)
+- `.mdx` (with the [MDX Integration installed](/en/guides/markdown-content/#mdx-features))
+- [`.js`/`.ts`](/en/core-concepts/astro-pages/#non-html-pages)
+- `.html` (following the same restrictions as [HTML Components](/en/core-concepts/astro-components/#html-components))
+
+
+## Astro Pages
+
+Astro pages use the `.astro` file extension and support the same features as [Astro components](/en/core-concepts/astro-components/).
+
 ### Page HTML
```

Comment:
Reading through this I’m struck that the heading “Page HTML” follows not long after we mention `.html` pages. Do you think that matters? Would renaming that heading help clarity at all or is it not an issue?

Reply:
I agree. I'd remove it entirely. It made more sense when we were really ONLY talking about Astro pages here.

Looking at the right sidebar now, I'm also considering whether that layouts subheading deserves to be there, too... will report back!

Reply:
“Required Page Content”?

Reply:
(ah, just saw the recommendation to delete — that works too!)